### ISSUE TITLE
Zip operator not working

### ISSUE THREAD

**Issue by OopsYao** [ReactiveX/RxPY#578]
The demo of `zip` operator in the [migration guide](https://rxpy.readthedocs.io/en/latest/migration.html#removal-of-the-result-mapper) does not work as expected.
```python
import rx
from rx import operators as ops
import operator

a = rx.of(1, 2, 3, 4)
b = rx.of(2, 2, 4, 4)

a.pipe(
    ops.zip(b), # returns a tuple with the items of a and b
    ops.map(lambda z: operator.mul(z[0], z[1]))
).subscribe(print)
```
Expected: print 2, 4, 12, 16 sequentially; actually there is no output.

OS: Arch Linux x86_64 with kernel 5.13.6-arch1-1
Python: 3.9.6
Rx: 3.2.0

**Comment by noflame** [comment]
The same thing happened to me here.
So I had to use version 3.1.1, the zip function works fine

os: Win10
Python 3.7.6

**Comment by elbowz** [comment]
+1

**Comment by MainRo** [comment]
Fixed in #595

**Comment by chunzhao** [comment]
Noob question: Is the fix of this issue going to be patched into RxPY v3.0? 
I'm relying on pip to install RxPY (pip3 install rx). The issue of zip operator exists in v3.2.0 as of today.

**Comment by HungryAfterLunch** [comment]
Some here. The zip operator work fine in v3.1.1 but not working on v3.2.0.

os: win11
python: 3.10.4
